internal/providers/terraform/google/testdata/bigquery_dataset_test/bigquery_dataset_test.usage.yml

```
@@ -0,0 +1,4 @@
+version: 0.1
+resource_usage: 
+  google_bigquery_dataset.usage:
+    monthly_queries_tb: 100
```

It would be nice to make this a decimal so we can see how it handles fractional TB.
```suggestion
    monthly_queries_tb: 100.3
```